src/environments/assets.ts

```diff
@@ -13,4 +13,16 @@ export const assets = [
   'assets/release-notes-1.15.2.json',
   'assets/release-notes-1.15.1.json',
   'assets/release-notes-1.15.json',
```

Comment:
It looks like that I've introduced an inconsistency for the minor releases with v1.17.0, see:
- `assets/release-notes-1.17.0.json`
- `assets/release-notes-1.15.json`
- `assets/release-notes-1.14.json`

Feel free to adapt this as well if you want to :)

Reply:
I can look at that while I'm in there. Which would we want to stick with `x.x.x` or `x.x`? My feeling is `x.x.x` but I can be easily swayed :)

Reply:
I agree that `x.x.x` might be a more consistent approach. :+1:

Reply:
I will make it so